src/com/datacash/training/chapter14/Question9.java

```diff
@@ -0,0 +1,29 @@
+package com.datacash.training.chapter14;
+
+interface StringFunc {
+  String func( String str );
+}
+
+public class Question9 { 
+	   
+	  static String changeStr(StringFunc sf, String s) { 
+	    return sf.func(s); 
+	  } 
+	 
+	  public static void main(String args[]) 
+	  { 
+	    String inStr = "Lambda Expressions Expand Java"; 
+	    String outStr; 
+	 
+	    System.out.println("Input string: " + inStr); 
+	 
+	    // Remove spaces
+	    StringFunc noSpaces = (str) -> str.replace(" ", ""); 

```

Comment:
Yeah, you'd be insane to write it like that (the way the book does it) .... I'm starting to think the author is insane.